tests/testdateparam.py

```diff
@@ -11,7 +11,7 @@
 
 try:
     import numpy as np
-except:
+except ImportError:
```

Comment:
What do you think about catching `ModuleNotFoundError` instead?

Reply:
I have update every ImportError in ccaf158 to ModuleNotFoundError